src/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebarExtraLinks.test.tsx

```diff
@@ -32,13 +33,30 @@ describe("ArtworkSidebarExtraLinks", () => {
     })
   }
 
+  describe("for work in a benefit auction", () => {
+    beforeAll(async () => {
+      wrapper = await getWrapper(BenefitAuctionArtwork)
+    })
+    it("displays proper conditions of sale text", () => {
+      expect(wrapper.text()).toContain(
+        "By placing your bid you agree to Artsy's Conditions of Sale."
+      )
+      expect(wrapper.text()).toContain(
+        "Have a question? Read our auction FAQs or ask a specialist."
+      )
+      expect(wrapper.text()).toContain(
+        "Want to sell a work by this artist? Consign with Artsy."
+      )
+    })
+  })
+
   describe("for work in an auction", () => {
     beforeAll(async () => {
       wrapper = await getWrapper(LiveAuctionArtwork)
     })
-    it("displays proper text", () => {
+    it("displays proper conditions of sale text", () => {
```

Comment:
Since you have a bit of logic around the partner name ending with `s`, it would be good to add a test exercising that case!